### PR TITLE
[explorer] - update logo / header + footer

### PR DESCRIPTION
**Pull request by mamos-mysten**
## Description 

Describe the changes or additions included in this PR.

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Comment by mamos-mysten** [comment]
Current dependencies on/for this PR:
* main
  * **PR #11511** <a href="https://app.graphite.dev/github/pr/MystenLabs/sui/11511" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/MystenLabs/sui/11511?utm_source=stack-comment).

File: apps/explorer/src/assets/SuiSymbol.svg
Comment (williamrobertson13) [inline apps/explorer/src/assets/SuiSymbol.svg:1]: don't we already have an icon in `@mysten/icons` for the Sui logo? and maybe potentially the word mark as well?

I think the wallet header uses them

File: apps/explorer/src/components/footer/Footer.tsx [hunk 1]
```diff
@@ -1,61 +1,131 @@
...
+            </li>
+            <li>
+                <Link variant="text" href="https://mystenlabs.com/#community">
+                    <Text variant="body/medium" color="steel-darker">
+                        Docs
+                    </Text>
+                </Link>
+            </li>
+            <li>
+                <Link variant="text" href="https://mystenlabs.com/#community">
```
Comment (williamrobertson13) [inline apps/explorer/src/components/footer/Footer.tsx:50]: this isn't a Github link 🥸

File: apps/explorer/src/components/footer/Footer.tsx [hunk 2]
```diff
@@ -1,61 +1,131 @@
...
+                    <Text variant="body/medium" color="steel-darker">
+                        Press
+                    </Text>
+                </Link>
+            </li>
+            <li>
+                <Link variant="text" href="https://mystenlabs.com/#community">
```
Comment (williamrobertson13) [inline apps/explorer/src/components/footer/Footer.tsx:43]: this isn't a docs link 🥸

File: apps/explorer/src/components/footer/Footer.tsx [hunk 3]
```diff
@@ -1,61 +1,131 @@
...
-                    <FooterLinks links={footerLinks} />
+
+                <div className="h-[1px] w-full bg-gray-45" />
+                <div className="flex w-full items-center justify-between">
+                    <div className="h-full space-y-2">
+                        <Text
+                            color="steel-darker"
+                            variant="pSubtitleSmall/medium"
+                        >
+                            &copy;
```
Comment (williamrobertson13) [inline apps/explorer/src/components/footer/Footer.tsx:96]: is this fine to keep on the legal side of things?

File: apps/explorer/src/components/footer/Footer.tsx [hunk 4]
```diff
@@ -1,61 +1,131 @@
...
+                            {`${new Date().getFullYear()} Sui. All
+                                rights reserved.`}
+                        </Text>
+                    </div>
+                    <ul className="flex  gap-2">
+                        <li>
+                            <Link
+                                variant="text"
+                                href="https://mystenlabs.com/#community"
```
Comment (williamrobertson13) [inline apps/explorer/src/components/footer/Footer.tsx:105]: wrong link 🥸

File: apps/explorer/src/components/footer/Footer.tsx [hunk 5]
```diff
@@ -1,61 +1,131 @@
...
+                                    color="steel-darker"
+                                >
+                                    Terms & Conditions
+                                </Text>
+                            </Link>
+                        </li>
+                        <li>
+                            <Link
+                                variant="text"
+                                href="https://mystenlabs.com/#community"
```
Comment (williamrobertson13) [inline apps/explorer/src/components/footer/Footer.tsx:118]: wrong link 🥸

File: apps/explorer/src/components/footer/footerLinks.ts [hunk 6]
```diff
@@ -1,65 +1,42 @@
...
 // SPDX-License-Identifier: Apache-2.0
 
-type FooterItem = {
-    category: string;
-    items: { title: string; href: string }[];
-};
+type FooterItem = { title: string; href: string }[];
+
 export type FooterItems = FooterItem[];
 export const footerLinks = [
```
Comment (williamrobertson13) [inline apps/explorer/src/components/footer/footerLinks.ts:7]: seems like we can delete this file with the new footer?

File: apps/explorer/src/components/footer/Footer.tsx (hunk 5 above)
Comment (mamos-mysten) [reply apps/explorer/src/components/footer/Footer.tsx:118]: whooops, forgot to push latest. sry for the premature request

File: apps/explorer/src/components/footer/Footer.tsx (hunk 3 above)
Comment (mamos-mysten) [reply apps/explorer/src/components/footer/Footer.tsx:96]: not sure what you mean here - the date? or the content of the text?

File: apps/explorer/src/assets/SuiSymbol.svg
Comment (mamos-mysten) [reply apps/explorer/src/assets/SuiSymbol.svg:1]: Yea, it was called out in slack that the `icons` app is supposed to be a 1:1 with the icons in Figma. The Sui wordmark doesn't exist there, and the SuiLogoTxt component doesn't contain the registered trademark symbol that we have in the designs.

File: apps/explorer/src/components/footer/footerLinks.ts (hunk 6 above)
Comment (mamos-mysten) [reply apps/explorer/src/components/footer/footerLinks.ts]: deleted

File: apps/explorer/src/components/footer/Footer.tsx (hunk 3 above)
Comment (williamrobertson13) [reply apps/explorer/src/components/footer/Footer.tsx:96]: oh sorry I meant the copyright for Sui, I wasn't sure if we were going to move the copyright to Mysten Labs or something with the ownership transfer

File: apps/explorer/src/assets/SuiSymbol.svg
[the remark this answers is not in the document]
Comment (mamos-mysten) [reply apps/explorer/src/assets/SuiSymbol.svg:1]: Oh yea, I replied: "Yea, it was called out in slack that the icons app is supposed to be a 1:1 with the icons in Figma. The Sui wordmark doesn't exist there, and the SuiLogoTxt component doesn't contain the registered trademark symbol that we have in the designs."

File: apps/explorer/src/components/footer/Footer.tsx (hunk 3 above)
Comment (mamos-mysten) [reply apps/explorer/src/components/footer/Footer.tsx:96]: this mirrors what we have on the Sui Foundation website: 
image

**Review by mystie711 — APPROVED** [review]
🚀

**Pull request merged**